tests/resources/test_conformance.py

```diff
@@ -54,7 +54,11 @@ async def test_landing_page_links(
     assert link.get("type") == expected_media_type
 
     link_path = urllib.parse.urlsplit(link.get("href")).path
-    assert link_path == app.state.router_prefix + expected_path
+
+    # this feels hacky
+    assert link_path == (
+        app.state.router_prefix + app.state.router_prefix + expected_path
+    ) or (app.state.router_prefix + expected_path)
```

Comment:
😓 yep that's def not good!

Reply:
let's look at this before we merge

Reply:
it's definitely a weird situation

Reply:
let me look

Reply:
ok :)

Reply:
there is definitely a bug, when we have `router_prefix="/router_prefix"` we end up having urls being `/router_prefix/router_prefix/api.html` instead of `/router_prefix/api.html`

Reply:
easy fix?

Reply:
looking

Reply:
😬 this is a stac-fastapi issue 

in https://github.com/stac-utils/stac-fastapi/blob/f54227871c29cf15ae49b2ab96b28f357de60485/stac_fastapi/types/stac_fastapi/types/requests.py#L6-L14 
we add the `prefix` to the `base_url` and in https://github.com/stac-utils/stac-fastapi/blob/f54227871c29cf15ae49b2ab96b28f357de60485/stac_fastapi/types/stac_fastapi/types/core.py#L614-L631

we do `base_url` + `openapi_url`, BUT when we init the application we already add the prefix to the `openapi_url` 😓 

we will need to fix stac-fastapi (and add test)

Reply:
introduced in https://github.com/stac-utils/stac-fastapi/pull/635

Reply:
This is really weird, the test should not work but for some reason we still get `/router_prefix/router_prefix/api` 😭

Reply:
yikes